docs/adr/adr-010-send-unpublish-emails-for-single-pages.md

```diff
@@ -0,0 +1,79 @@
+
+### Addressing the concerns from the previous version
+
+The previous version of this feature was removed for several good reasons. We believe readding the unpublish emails only for single page subscriptions addresses these concerns:
+
+#### It only worked for topic taxons
+
+Single page notification work generates a clearer user need to inform about subscriptions in the more frequent scenario that individual pages are unpublished.
+
+Given the specificity of the need that drives this, we don't think limiting it to just those subscriptions is a problem.
+
+#### It only worked if the unpublished page was redirected
+
+The new unpublished emails will handle pages that are redirected or published in error with or without an alternative URL. This covers the publisher-facing [unpublishing types][unpublishing-types].
+
+#### It wasn't monitored so we didn't know if it was working
+
+We don't have a specific plan for monitoring this. Ideally monitoring for this would be taken on by a team that owns this portion of the system in the log run, and incorporated into their usual service monitoring.
+
+#### It rendered email using ERB, unlike the rest of the system
+
+The new unpublished emails will use the same method to render emails as the content changes.
+
+#### It removed any subscriber lists that included the page, even those for a different taxon
+
+We are taking a very different approach from the previous unpubication of taxons. The previous route had to infer which lists should be removed as they represented a collection of materials.
+
+Here we are unpublishing a single page, and matching it to a subscription list by the same content ID. The email alert api doesn't need to do anything clever to match the two, we just tell it which subscription list should be cleared, and it does so.
+
+#### There was no evidence it met user needs
+
+Single page notifications are an answer to the user need to stay up to date with specific pieces of government guidance. Especially when the area might be subject to frequent updates or changes.
+
+Our hypothesis is that some users will subscribe to taxons that are broader than their actual area of interest. So they will receive updates about areas they are not interested in, in order to get the few relevant updates to them.
+
+Our hypothesis is that users who subscribe to be notified for major updates, will also want to know if the guidance is unpublished as part of staying up-to-date, and will find it unexpected if guidance they have been monitoring disappears from their account subscriptions without warning.
+
+### Other related issues
+
+The email alert API has subscriber lists that will never send an email. This is documented as [GOV.UK Tech Debt][email-alert-api-dead-lists].
+This proposal does not directly address this Tech Debt, but by removing the single page subscriber lists as a page is unpublished we can try to prevent the impact of the tech debt from increasing.
```

Comment:
I think we need to update this line now the email alert API endpoint no longer deletes the subscriber list.

Perhaps:
```suggestion
This proposal does not directly address this Tech Debt, but by removing subscriptions from the subscriber list once the emails have been sent the [HistoricalDataDeletionWorker](https://github.com/alphagov/email-alert-api/blob/main/app/workers/historical_data_deletion_worker.rb) can periodically remove the empty lists.
```